### ISSUE TITLE
[Bug]: mo panic with not found when run stability test

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#5805]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.6.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

{"level":"INFO","time":"2022/10/20 23:13:41.712596 +0800","caller":"compile/scope.go:43","msg":":0  union all  -> sql output"}
{"level":"INFO","time":"2022/10/20 23:13:41.712605 +0800","caller":"frontend/mysql_cmd_executor.go:439","msg":"goid 914479 \n"}
{"level":"INFO","time":"2022/10/20 23:13:41.712835 +0800","caller":"frontend/mysql_cmd_executor.go:524","msg":"rowCount 1 \ntime of getDataFromPipeline : 24.223µs \nprocessBatchTime 14.879µs \nrow2colTime 0s \nallocateOutBufferTime 2.052µs \noutputQueue.flushTime 0s \nprocessBatchTime - row2colTime - allocateOutbufferTime - flushTime 12.827µs \nrestTime(=tTime - row2colTime - allocateOutBufferTime) 22.171µs \nprotoStats flushCount 0 writeCount 1 \nwriteBytes 0 0 MB\n\n"}
{"level":"INFO","time":"2022/10/20 23:13:41.686677 +0800","caller":"indexwrapper/immutindex.go:82","msg":"error: tae possible duplicate","span":{}}
{"level":"INFO","time":"2022/10/20 23:13:41.708820 +0800","caller":"frontend/mysql_cmd_executor.go:2627","msg":"cmd 3"}
{"level":"INFO","time":"2022/10/20 23:13:41.713857 +0800","caller":"frontend/mysql_cmd_executor.go:2642","msg":"query trace","connection_id":1007,"query":"select c from sbtest1 where id = 25383;"}
{"level":"INFO","time":"2022/10/20 23:13:41.714076 +0800","caller":"frontend/mysql_cmd_executor.go:439","msg":"goid 917280 \n"}
{"level":"INFO","time":"2022/10/20 23:13:41.714251 +0800","caller":"frontend/mysql_cmd_executor.go:524","msg":"rowCount 1 \ntime of getDataFromPipeline : 12.17µs \nprocessBatchTime 6.056µs \nrow2colTime 0s \nallocateOutBufferTime 1.373µs \noutputQueue.flushTime 0s \nprocessBatchTime - row2colTime - allocateOutbufferTime - flushTime 4.683µs \nrestTime(=tTime - row2colTime - allocateOutBufferTime) 10.797µs \nprotoStats flushCount 0 writeCount 1 \nwriteBytes 0 0 MB\n\n"}
{"level":"INFO","time":"2022/10/20 23:13:41.714529 +0800","caller":"db/db.go:84","msg":"error: not found","span":{}}
panic: not found

goroutine 3869368 [running]:
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/moengine.(*txnEngine).Database(0xc438a9ee40?, {0x2848878, 0xc0cf2793b0}, {0xc05c9b76e4, 0x5}, {0x28752c8?, 0xc0ddc43470?})
        /data1/sudong/mo-regression/matrixone/pkg/vm/engine/tae/moengine/engine.go:130 +0x107
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).ParallelRun(0xc0dda78be0, 0xc017ea7860)
        /data1/sudong/mo-regression/matrixone/pkg/sql/compile/scope.go:151 +0x99
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).RemoteRun(0x1eb1f46e0?, 0xc000708168?)
        /data1/sudong/mo-regression/matrixone/pkg/sql/compile/scope.go:127 +0x125
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func3(0x0?)
        /data1/sudong/mo-regression/matrixone/pkg/sql/compile/scope.go:89 +0x75
created by github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun
        /data1/sudong/mo-regression/matrixone/pkg/sql/compile/scope.go:84 +0x58d
~

**Comment by LeftHandCold** [comment]
fixed

**Comment by LeftHandCold** [comment]
It is possible in which pr it was fixed

**Comment by aressu1985** [comment]
fixed:249f56400184a0828afdc615c8b5d75e8c57cecc
